node/install.sh

```diff
@@ -9,5 +9,5 @@ then
     fi
 fi
 
-
-
+# install global libraries
+pnpm i -g dotenv-cli
```

Comment:
@kipprice-mbh , I'm not sure if this would work on a clean setup. You might have to check if `pnpm` is installed, like we do on line #3. I'm not suggesting any changes, let's just keep this in mine for when we set Michael's environment.